src/main/java/org/broadinstitute/hellbender/tools/examples/metrics/multi/ExampleMultiMetricsCollectorSpark.java

```diff
@@ -0,0 +1,101 @@
+package org.broadinstitute.hellbender.tools.examples.metrics.multi;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.metrics.Header;
+import htsjdk.samtools.metrics.MetricsFile;
+import org.apache.spark.api.java.JavaRDD;
+import org.broadinstitute.hellbender.engine.AuthHolder;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.pipelines.metrics.MetricsCollectorSpark;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Example implementation of a multi-level Spark metrics collector. Can be embedded within a standalone tool
+ * (see @link ExampleCollectMultipleMetricsSpark}, a pipeline, or
+ * {@link org.broadinstitute.hellbender.tools.spark.pipelines.metrics.CollectMultipleMetricsSpark}.
+ */
+public class ExampleMultiMetricsCollectorSpark
+        implements MetricsCollectorSpark<ExampleMultiMetricsArgumentCollection>,
+        Serializable
+{
+    private static final long serialVersionUID = 1L;
+
+    private ExampleMultiMetricsArgumentCollection inputArgs = null;
```

Comment:
Its isn't necessary but only because there are no interesting arguments for the example. I'd like to leave it to illustrate the pattern, but will add a comment clarifying that.